identifiers/country-in/vidhan_sabha.csv

```diff
@@ -1,771 +1,4034 @@
-ocd-division/country:in/state:mh/district:th/cd:kalyan__west,Kalyan West constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:kalyan_east,Kalyan East constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:kalyan_rural,Kalyan Rural constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:kopripachpakhadi,Kopripachpakhadi constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:meera_bhayandar,Meera Bhayandar constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:mumbrakalwa,Mumbrakalwa constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:murbad,Murbad constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:ovala_majiwada,Ovala Majiwada constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:shahapur,Shahapur constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:thane,Thane constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:th/cd:ulhasnagar,Ulhasnagar constituency; Thane district; Maharashtra
-ocd-division/country:in/state:mh/district:wr/cd:arvi,Arvi constituency; Wardha district; Maharashtra
-ocd-division/country:in/state:mh/district:wr/cd:deoli,Deoli constituency; Wardha district; Maharashtra
-ocd-division/country:in/state:mh/district:wr/cd:hinganghat,Hinganghat constituency; Wardha district; Maharashtra
-ocd-division/country:in/state:mh/district:wr/cd:wardha,Wardha constituency; Wardha district; Maharashtra
-ocd-division/country:in/state:mh/district:ws/cd:karanja,Karanja constituency; Washim district; Maharashtra
-ocd-division/country:in/state:mh/district:ws/cd:risod,Risod constituency; Washim district; Maharashtra
-ocd-division/country:in/state:mh/district:ws/cd:washim,Washim constituency; Washim district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:arni,Arni constituency; Yavatmal district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:digras,Digras constituency; Yavatmal district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:pusad,Pusad constituency; Yavatmal district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:ralegaon,Ralegaon constituency; Yavatmal district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:umarkhed,Umarkhed constituency; Yavatmal district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:wani,Wani constituency; Yavatmal district; Maharashtra
-ocd-division/country:in/state:mh/district:yt/cd:yavatmal,Yavatmal constituency; Yavatmal district; Maharashtra
\ No newline at end of file
+id,name,
+ocd-division/country:in/state:ap/district:an/cd:anantapur_urban,Anantapur Urban; Anantapur,

```

Comment:
Please omit the district name (`; Anantapur`) from the constituency name.

Reply:
Done